docs/source/generic/rest.rst

```diff
@@ -43,7 +43,7 @@ Paginering
 ----------
 
 De resultaten worden gepagineerd teruggegeven.
-De paginagrootte kan aangepast worden door een :samp:`?page_size={n}` query parameter toe te voegen aan de request URL.
+De paginagrootte kan aangepast worden door een :samp:`?pageSize={n}` query parameter toe te voegen aan de request URL.
```

Comment:
Mooi, het moet trouwens deze zijn `_pageSize`

Reply:
idd, de underscore moet ervoor: het is een "escape" van de normale filter parameters (die zonder _ zijn)

Reply:
Oeps, te snel geweest met mergen. #202.